src/navigation/navigation.tsx

```diff
@@ -943,6 +954,22 @@ export const FunkeC2ShareStackScreenWithContext = (props: any): JSX.Element => (
   </FunkeC2ShareProvider>
 );
 
+const ShareStack = (): JSX.Element => {
+  return (
+    <ShareBaseStack.Navigator screenOptions={{animation: 'none'}} initialRouteName={ScreenRoutesEnum.QR_PRESENTATION}>
+      <ShareBaseStack.Screen
+        name={ScreenRoutesEnum.QR_PRESENTATION}
+        component={QRPresentationScreen}
+        options={{
+          header: props => <SSIHeaderBar headerSubTitle="Show the QR code to the party of interest. So they can scan your QR code." {...props} />,
+          headerTitle: 'Present QR code',
+          headerShown: true,
```

Comment:
can be omitted, default is true

Reply:
https://reactnavigation.org/docs/stack-navigator/#headershown

Reply:
I have removed this `headerShown` option from the object